### PR TITLE
Single tech/databricks 197 sample2 readme

### PR DESCRIPTION
**Pull request by namitms**
## Purpose
* Read-me for Sample 2

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

## Author pre-publish checklist:
- [ ] Added test to prove my fix is effective or new feature works
- [x] No PII in logs
- [x] Made corresponding changes to the documentation

Closing #197

### CODE REVIEW

File: single_tech_samples/databricks/sample2_enterprise_azure_databricks_environment/README.md [hunk 1]
```diff
@@ -1 +1,269 @@
...
+>
+![alt text](../Common_Assets/Images/ADE_PL.png "Logo Title Text 1")
+
+### 3.4. Deployment validation
+
+The following steps can be performed to validate the correct deployment of this sample:
+
+1. Users with appropriate access rights should be able to:
+
+   1. launch the workspace from the Azure portal.
```
Comment (devlace) [inline single_tech_samples/databricks/sample2_enterprise_azure_databricks_environment/README.md:248]: Nit: Capitalize "l"

File: single_tech_samples/databricks/sample2_enterprise_azure_databricks_environment/README.md [hunk 2]
```diff
@@ -1 +1,269 @@
...
+
+5. Use [Azure Policies](https://azure.microsoft.com/en-us/services/azure-policy/) to define guardrails around deployment constraints to regulate the cost
+
+### 2.2. Operational Excellence
+
+1. Ensure that the parameters passed to the deployment scripts are validated
+
+1. Leverage parallel resource deployment where ever possible. In the scope of this sample, all three resources can be deployed in parallel.
+
+1. Validate compensation transactions for the deployment workflow.
```
[the remark this answers is not in the document]
Comment (namitms) [reply single_tech_samples/databricks/sample2_enterprise_azure_databricks_environment/README.md:114]: In the context of this sample, the idea is to tear down partially provisioned resources if the provisioning fails

**Comment by namitms** [comment]
> The WAF section can use a bit more work, but for now i think its a good start. We can merge the current content then do another round to update the WAF sections.
> 
> A bit of nit: sometimes end points with periods and sometimes you don't. Would be good to make it consistent. :)

Yes, I just copied it from sample 1 :). We can work on this post merge if that is ok with you

**Comment by devlace** [comment]
LGTM

**Review by devlace — APPROVED** [review]

**Pull request merged**